app/poros/image_options.rb

```diff
@@ -0,0 +1,12 @@
+class ImageOptions
+  attr_reader :deep_id, :pixabay_id, :deep_url, :pixabay_url, :deep_source, :pixabay_source
+
+  def initialize(info)
```

Comment:
When a JSON payload is retrieved from our Sinatra microservice, it is in the same format every time. That allows us to use [0] and [1] to initialize ImageOptions.

Reply:
🔥